src/main/java/manageezpz/logic/commands/EditTaskCommand.java

```diff
@@ -118,20 +122,73 @@ public CommandResult execute(Model model) throws CommandException {
         }
```

Comment:
In line 121 of ur new code, can you add the throwable exception behind? It is not necessary but I realised that I missed it out.

```
catch (DuplicateTaskException de) {
    throw new CommandException(String.format(MESSAGE_DUPLICATE_TASK, this.desc) + MESSAGE_USAGE, de);
}
```
This exception will happen if we have a task with the same desc in the task list.